tsl/generatePromQl.go

```diff
@@ -261,7 +262,7 @@ func (protoParser *ProtoParser) promSampleBy(sampleBy FrameworkStatement) (strin
 }
 
 // Generate each individual method statement
-func (protoParser *ProtoParser) promFrameworksOp(frameworks []FrameworkStatement, promql *Ql, skipSample bool) (string, error) {
+func (protoParser *ProtoParser) promFrameworksOp(frameworks []FrameworkStatement, promql *Ql, skipSample bool) (string, bool, error) {
```

Comment:
Perhaps a little comment to explain what this function returns (string, bool, error)

Reply:
Done, thanks :)